scipy/optimize/tests/test__shgo.py

```diff
@@ -406,18 +425,51 @@ def test_f3_simplicial(self):
 
     @pytest.mark.slow
     def test_f4_simplicial(self):
-        """NLP: (High-dimensional) Hock and Schittkowski 11 problem (HS11)"""
+        """NLP: (High dimensional) Hock and Schittkowski 11 problem (HS11)"""
         run_test(test4_1, n=1, sampling_method='simplicial')
 
-    def test_lj_symmetry(self):
+    @pytest.mark.xfail(reason="backward incompatible change")
+    def test_lj_symmetry_old(self):
         """LJ: Symmetry-constrained test function"""
-        options = {'symmetry': True,
+        options = {'symmetry': [True],
```

Comment:
> This is a mistake in the documentation, it should be bool or list. When specifying a bool the behaviour is identical and should be backwards-compatible. Is this ok?

Oops and it looks lIke I made a mistake in resurrecting this test, which had been removed.  I should have done:
```suggestion
        options = {'symmetry': True,
```
If this passes, then `bool` and `list` is fine, no need for `bool_list`. Please include this (and remove the `xfail`) when you make other changs.